apps/codeimage/src/components/ThemeSwitcher/PresetSwitcher.tsx

```diff
@@ -194,6 +195,26 @@ export const PresetSwitcher: ParentComponent<
                             <div>
                               <DropdownMenuV2
                                 onAction={(action: string | number) => {
+                                  if (action === 'update') {
+                                    createDialog(PresetUpdateDialog, state => ({
```

Comment:
@riccardoperra wytd if we refactor creating a record instead of using these conditions?